test/tests/caliptra_integration_tests/smoke_testdata/rt_alias_cert_redacted.txt

```diff
@@ -30,7 +30,7 @@ Certificate:
             2.23.133.5.4.4: 
                 0....................
             2.23.133.5.4.1: 
-                DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD
+                DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD
```

Comment:
What's the change that leads to the TCBInfo extension being shorter?

Reply:
This is the removal of Vendor="Caliptra" and Model="RT" from this info.